Lib/test/test_capi/test_abstract.py

```diff
@@ -339,6 +339,20 @@ def test_object_delattrstring(self):
         # CRASHES delattrstring(obj, NULL)
         # CRASHES delattrstring(NULL, b'a')
 
+    def test_copy_to_object(self):
+        copy_to_object = _testcapi.object_copy_to_object
+        s1 = copy_to_object(bytearray(3), 'abc', b'C')
+        s2 = copy_to_object(bytearray(3), 'abc', b'F')
+        s3 = copy_to_object(bytearray(3), 'abc', b'A')
+        self.assertEqual(s1, s2)
```

Comment:
I would prefer to compare it to a known bytearray, such as: `bytearray(b'abc')`.